x11-edge-grab/src/config.rs

```diff
@@ -0,0 +1,64 @@
+use std::path::Path;
+use std::fs::File;
+use std::io::Read;
+
+use serde_yaml;
+
+#[derive(Serialize, Deserialize, Debug, Clone)]
+pub struct Config {
+    pub monitors: Vec<Monitor>,
+}
+
+#[derive(Serialize, Deserialize, Debug, Clone, PartialEq, Eq)]
+pub struct Monitor {
+    pub bounds: Rectangle,
+    pub is_windows: bool,
+    /// whether this monitor is still connected after switching inputs to Windows
+    #[serde(default)]
+    pub connected: bool,
+}
+
+#[derive(Serialize, Deserialize, Debug, Clone, PartialEq, Eq)]
+pub struct Rectangle {
+    pub x: i32,
+    pub y: i32,
```

Comment:
I think that a rectangle consists either of 2 points, or x,y,w,h. And it's shorter to access this way ;)